internal/schemautil/service.go

```diff
@@ -624,7 +624,7 @@ func copyServicePropertiesFromAPIResponseToTerraform(
 		return fmt.Errorf("cannot set `components` : %s", err)
 	}
 
-	return copyConnectionInfoFromAPIResponseToTerraform(d, serviceType, s.ConnectionInfo)
+	return copyConnectionInfoFromAPIResponseToTerraform(d, serviceType, s.ConnectionInfo, s.Metadata)
```

Comment:
Can we add a test that checks that we get this property set?

Reply:
`s.Metadata` always exists in every API response of this type, it also always exists in the Go client, is there really need for a test?

Reply:
I mean we can just add this check to one of the existing tests